docker/repositoryserver-controller/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.1.0-1656.1669627757
```

Comment:
I see this was updated recently. Let's make sure we update to latest before merging.